kotlin/internal/jvm/impl.bzl

```diff
@@ -34,7 +34,7 @@ def _make_providers(ctx, providers, transitive_files = depset(order = "default")
             providers.java,
             providers.kt,
             DefaultInfo(
-                files = depset([ctx.outputs.jar]),
+                files = depset([ctx.outputs.jar, ctx.outputs.jdeps]),
```

Comment:
Does this need to be in the default info for the IDE? I'd prefer to leave it as a "build on request" artifact. Also should it go into JavaInfo?

Reply:
Done - for some reason I couldn't get it working without this, but I just removed it... and its working :-)